packages/terra-list/tests/wdio/list-sections-spec.js

```diff
@@ -0,0 +1,59 @@
+/* global browser, Terra */
+
+describe('List with Sections', () => {
```

Comment:
Should we test these in different viewports?

Reply:
In Nightwatch, all the tests are run against, Medium Size - [Link](https://github.com/cerner/terra-core/blob/master/packages/terra-list/tests/nightwatch/multi-select-list-spec.js#L9), so I thought, its okay to test it in default viewport. Should I be testing it in all the viewports for WDIO?